ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -965,8 +989,9 @@ export default class ConfirmTransactionBase extends Component {
         lastTx={lastTx}
         ofText={ofText}
         requestsWaitingText={requestsWaitingText}
+        hideConfirmAnyways={!isDisabled()}
         disabled={
-          !valid ||
+          isDisabled() ||
           submitting ||
```

Comment:
[nit] What do you think about placing the rest of these in `isDisabled` as well? It might be easier from a maintenance perspective if these were all in one place rather than split up.

Reply:
isDisabled() is used elsewhere that doesn't require the extra checks.